src/nfs.c

```diff
@@ -596,6 +612,34 @@ static int nfs_read(void) {
 }
   /* #endif KERNEL_LINUX */
 
+#elif KERNEL_NETBSD
+static int nfs_read(void) {
+  struct nfsstats ns;
+  size_t size = sizeof(ns);
+  int mib[] = {CTL_VFS, 2, NFS_NFSSTATS};
+  value_t values[nfs3_procedures_names_num];
+  int i;
+
+  /* NetBSD reports v2 statistics mapped to v3 and doen't yet support v4 */
+  if (!report_v3)
+    return 0;
+  if (sysctl(mib, 3, &ns, &size, NULL, 0) != 0)
```

Comment:
I would use STATIC_ARRAY_SIZE(mib) for length to avoid magic numbers.

Reply:
Updated.